tree/treeplayer/src/TMPWorkerTree.cxx

```diff
@@ -267,11 +307,12 @@ void TMPWorkerTreeSel::Process(unsigned int code, MPCodeBufPair& msg)
       unsigned nEntries = fTree->GetEntries();
```

Comment:
I understand this are pre-existing ... but they are fatally wrong.  They must be 64 bits integer.   GetEntries returns a Long64_t.